src/player.cpp

```diff
@@ -826,8 +827,14 @@ void Player::CreateGameObjects() {
 		if (!FileFinder::Game().FindFile("accord.dll").empty()) {
 			game_config.patch_maniac.Set(true);
 		}
+
+		if (!FileFinder::Game().FindFile(DESTINY_DLL).empty()) {
+			game_config.patch_destiny.Set(true);
+		}
 	}
 
+	Output::Debug("Patch configuration: dynrpg={} maniac={} key-patch={} common-this={} pic-unlock={} 2k3-commands={} destiny={}",
```

Comment:
This ``Output::Debug`` was removed and relocated to https://github.com/EasyRPG/Player/blob/2cb2956b055907ca5adbe842adec6f19261e502b/src/game_config_game.cpp#L234

Simply add a ``add_bool(patch_destiny);`` their